packages/sampleOS/build.yaml

```diff
@@ -13,6 +13,7 @@ join:
   name: "policy"
   version: ">=0"
 
+requires_final_images: true
```

Comment:
`join` is going away, `requires_final_images` actually indicates if using final images during build